app/src/main/java/edu/uc/willi6jd/mycaloriecounter/ui/main/foodView/FoodActivity.kt

```diff
@@ -15,13 +15,13 @@ import edu.uc.willi6jd.mycaloriecounter.R
  * This will control the view when they get to the food page
  */
 class FoodActivity : AppCompatActivity() {
-    var foodList = ArrayList<Food>()
```

Comment:
This array list is not used at the moment so I commented it out to let the developers know.

Reply:
Remove commented out code entirely.  Instead of commenting out code, my suggestion is:
1) Use feature branches in Git to keep experimental work separate from master.
2) If you're removing something that used to be there, just remove it entirely.  You can always go back in history to see what used to be there.